lms/djangoapps/course_home_api/progress/v1/views.py

```diff
@@ -168,6 +169,9 @@ def get(self, request, *args, **kwargs):
             'status_date': verification_status['status_date'],
         }
 
+        course_overview = CourseOverview.get_from_id(course_key)
+        enrollment = CourseEnrollment.get_enrollment(request.user, course_key)
```

Comment:
Do we still need line 132 given this? Feels like we might be fetching the same information twice.

Reply:
We don't, good catch!